scripts/fonts.json

```diff
@@ -0,0 +1,450 @@
+{
+  "font-families": {
+    "Gothic A1": ["regular", "700"],
+    "M PLUS Rounded 1c": ["regular", "700"],
+    "Noto Sans": ["regular", "700", "italic", "700italic"],
+    "Noto Sans Arabic": ["regular", "700"],
+    "Noto Sans Armenian": ["regular", "700"],
+    "Noto Sans Bengali": ["regular", "700"],
+    "Noto Sans Devanagari": ["regular", "700"],
+    "Noto Sans Ethiopic": ["regular", "600"],
+    "Noto Sans Georgian": ["regular", "700"],
+    "Noto Sans Gurmukhi": ["regular", "700"],
+    "Noto Sans Hebrew": ["regular", "700"],
+    "Noto Sans Kannada": ["regular", "700"],
+    "Noto Sans Lao": ["regular", "700"],
+    "Noto Sans Malayalam": ["regular", "700"],
+    "Noto Sans Myanmar": ["regular", "700"],
+    "Noto Sans Tamil": ["regular", "700"],
+    "Noto Sans Thaana": ["regular", "700"],
+    "Noto Sans Tifinagh": ["regular"],
+    "Noto Sans Telugu": ["regular", "700"],
+    "Noto Sans Thai": ["regular", "700"]
+  },
+  "bundle-font-stacks": {
+    "Noto Sans HK": ["regular", "700"],
+    "Noto Sans JP": ["regular", "700"],
+    "Noto Sans KR": ["regular", "700"],
+    "Noto Sans SC": ["regular", "700"],
+    "Noto Sans TC": ["regular", "700"]
+  },
```

Comment:
I’m pretty sure the user will never see these fonts, because GL JS is configured to use the browser’s default “sans-serif” CJK font by default: #613.

Reply:
I included these in order to allow a developer working on Han Unification to have multiple CJK fonts already rendered to work with.  They are not bundled into any of the Americana-* fontstacks, they're each rendered out to their own fontstack.

Reply:
OK, these are by far the largest fonts that would be downloaded when installing the package, but I guess we can live with that as long as it only happens once.

Reply:
A solution regarding Han Unification should probably retain the current client-side font solution rather than switching to server-side fontstacks, which carry significant performance penalties. (Aside from wasteful bandwidth usage, it’s very easy to hog memory and overflow buffers just by looking at East Asia when using server-side CJK fontstacks.)

The client-side font infrastructure is already mostly ready for deunified CJK fonts, except that GL JS only maintains a single instance of TinySDF that it initializes as soon as it needs to render any glyphs, and TinySDF only applies fonts to the canvas once at initialization. I suspect that addressing either limitation would be trivial.

Reply:
If it's not useful, I can drop these fonts.